2021-04-19 15:08:18,145 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>获取access_token
2021-04-19 15:08:40,260 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='api.21szxy.com', port=80): Max retries exceeded with url: /api/auth/access_token?app_id=client_id&app_secret=client_secret (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003B8FBE0>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:08:40,710 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>登陆参数正常-成功
2021-04-19 15:08:40,770 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:40,771 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/user/login?
2021-04-19 15:08:40,771 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "account": "11025688",
    "password": "QWER1357yj",
    "baseid": 0,
    "schoolid": 461361
}
2021-04-19 15:08:40,772 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:40,772 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:40,772 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:40,774 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'登陆成功' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:40,810 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>登陆参数为空-失败
2021-04-19 15:08:40,842 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:40,842 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/user/login?
2021-04-19 15:08:40,843 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "account": "",
    "password": "QWER1357yj",
    "baseid": 0,
    "schoolid": 461361
}
2021-04-19 15:08:40,843 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:40,843 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:40,843 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:40,843 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'传如参数有误' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:40,861 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>登陆参数错误-失败
2021-04-19 15:08:40,892 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:40,892 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/user/login?
2021-04-19 15:08:40,893 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "account": 110256880,
    "password": "QWER1357yj",
    "baseid": 0,
    "schoolid": 461361
}
2021-04-19 15:08:40,893 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:40,893 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:40,893 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:40,893 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'账号或密码错误' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:40,909 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>根据学号获取学生信息
2021-04-19 15:08:40,939 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:40,939 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/school/queryStudentByStuNo?
2021-04-19 15:08:40,939 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "schoolId": 461361,
    "schoolTerm": 1,
    "schoolYear": 2020,
    "studentNos": 210118001
}
2021-04-19 15:08:40,940 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:40,940 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:40,940 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:40,940 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'请求接口成功!' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:40,957 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>根据学年学期获取学生信息
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/school/getAllSchoolClassStudent?
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "choolTerm": 1,
    "schoolYear": 2020,
    "schoolId": 461361
}
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:40,988 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:40,989 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'请求接口成功!' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:41,006 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>获取所有学生或者获取当前学年学期的所有学生
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/school/getAllSchoolClassStudent
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "schoolId": 461361,
    "schoolTerm": 2,
    "schoolYear": 2020
}
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:41,037 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:41,038 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'请求接口成功!' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:41,056 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>根据班级获取学生信息
2021-04-19 15:08:41,086 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:121] - INFO: 请求方式====>GET
2021-04-19 15:08:41,087 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:122] - INFO: 请求地址====>http://api.21szxy.com/api/school/queryStudentByClassId?
2021-04-19 15:08:41,088 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:125] - INFO: 请求参数====>{
    "access_token": "None",
    "classIds": "81ab704b2e7544f4940b53f86727f5e2",
    "schoolId": 461361
}
2021-04-19 15:08:41,088 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:132] - INFO: 响应状态码为====>200
2021-04-19 15:08:41,089 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:133] - INFO: 接口响应头为====>{'Server': 'nginx/1.12.1', 'Date': 'Mon, 19 Apr 2021 07:08:40 GMT', 'Content-Type': 'text/JavaScript;charset=UTF-8', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Application-Context': 'ms-bd:8081', 'Nginx-Cache': 'BYPASS', 'Content-Language': 'zh-CN'}
2021-04-19 15:08:41,089 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:134] - INFO: 接口响应内容====>{"message":"解密token异常,无效的token!:None","code":-1}
2021-04-19 15:08:41,090 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:89] - ERROR: 断言： 预期值：'请求接口成功!' 实际值：'解密token异常,无效的token!:None', 断言失败!
2021-04-19 15:08:41,134 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>登陆首页
2021-04-19 15:09:02,176 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjsso.21cnjy.com', port=80): Max retries exceeded with url: /Account/Login?returnurl=http%253a%252f%252fyjadmin.21cnjy.com%252f%3A10192 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000003FB1D00>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:09:02,351 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>请求登陆
2021-04-19 15:09:23,352 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:59] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjsso.21cnjy.com', port=80): Max retries exceeded with url: /Account/Login?returnurl=http:%2F%2Fyjadmin.21cnjy.com:10192 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000000041EC7C0>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:09:23,514 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>设置Token
2021-04-19 15:09:44,513 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjsso.21cnjy.com', port=80): Max retries exceeded with url: /Auth/GetToken?returnurl=http%253A%252F%252Fyjadmin.21cnjy.com%3A10192%252F%253FToken%253D%2524Token%2524 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000000004199FD0>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:09:44,685 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>阅卷首页
2021-04-19 15:10:07,798 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjadmin.21cnjy.com', port=10192): Max retries exceeded with url: /Home/TeacherIndex (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000000042DFD60>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:10:07,943 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>任务列表
2021-04-19 15:10:28,945 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjadmin.21cnjy.com', port=10192): Max retries exceeded with url: /ExamOnline/List (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000000041A2BE0>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:10:29,074 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>阅卷界面
2021-04-19 15:10:50,121 - D:\PyCharm\Project\API_\API_Yaml\Basic\testBase.py[line:38] - ERROR: 接口请求异常,原因：HTTPConnectionPool(host='yjonline.21cnjy.com', port=80): Max retries exceeded with url: /Check/CommonReview?questionGroupId=None%5B0%5D&type=1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000000000462E400>: Failed to establish a new connection: [WinError 10060] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。'))
2021-04-19 15:10:50,283 - D:\PyCharm\Project\API_\API_Yaml\testcase\test_case.py[line:35] - INFO: 用例描述====>获取试卷
